src/plugins/dashboard/public/dashboard_container/dashboard_container_renderer.tsx

```diff
@@ -31,12 +31,14 @@ export interface DashboardContainerRendererProps {
   savedObjectId?: string;
   getCreationOptions?: () => DashboardCreationOptions;
   onDashboardContainerLoaded?: (dashboardContainer: DashboardContainer) => void;
+  showPlainSpinner?: boolean;
 }
 
 export const DashboardContainerRenderer = ({
   savedObjectId,
   getCreationOptions,
   onDashboardContainerLoaded,
+  showPlainSpinner,
```

Comment:
Instead of passing this in as a prop, can we subscribe to the observable _inside_ the `dashboard_container_renderer`? When it comes to our building blocks we try to get services as closely as possible to where they're being used / minimize the number of props.